.github/workflows/tests.yml

```diff
@@ -52,6 +56,7 @@ jobs:
           restore-keys: ${{ runner.os }}-sonar
 
       - name: Check formatting with spotless
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
Instead of this if condition, we can probably set `core.eol` to `lf` in a `.gitattributes` file so that spotless doesn't change the line endings while formatting. This should be done when we decide that sorald should support development on Windows too.

Reply:
That's not a full solution, as if we fully support Windows, we also need to have files with CRLF line endings in our test resources. To test the sniper printer output, that is. We literally need Git _not_ to change line endings or the tests break.

That being said, we could of course just set `core.autocrlf false` in a checked-in `.gitattributes` file.

Reply:
> we also need to have files with CRLF line endings in our test resources. To test the sniper printer output, that is.

Right. I did not consider this. :sweat_smile: